pkg/bbgo/standard_indicator_set.go

```diff
@@ -142,6 +142,7 @@ func (s *StandardIndicatorSet) BOLL(iw types.IntervalWindow, bandWidth float64)
 	if !ok {
 		inc = &indicator.BOLL{IntervalWindow: iw, K: bandWidth}
 		s.initAndBind(inc, iw.Interval)
+		inc.SMA = &indicator.SMA{IntervalWindow: iw}
```

Comment:
do we need this?

because:

```
func (inc *BOLL) Update(value float64) {
	if inc.SMA == nil {
		inc.SMA = &SMA{IntervalWindow: inc.IntervalWindow}
	}
```

Reply:
yes
or we have no SMA until the forst update

Reply:
ok~ why not directly embed into the constructor literal?

Reply:
done